lib/rules/unit-disallowed-list/__tests__/index.js

```diff
@@ -388,6 +388,64 @@ testRule({
 	],
 });
 
+testRule({
+	ruleName,
+
+	config: [
+		['px'],
+		{
+			ignoreFunctions: ['calc', /^translate/],
+		},
+	],
+
+	accept: [
+		{
+			code: 'a { margin: calc(50% - 100px) }',
+		},
+		{
+			code: 'a { transform: translate(50% - 100px, 100px) }',
+		},
+		{
+			code: 'a { transform: translateX(50% - 100px) }',
```

Comment:
[suggest] Same here.

```suggestion
			code: 'a { transform: translate(100px, 100px) }',
		},
		{
			code: 'a { transform: translateX(100px) }',
```